### PR TITLE
updated README.me

### PR DESCRIPTION
**Pull request by ciel**
I updated the README to include the text from [Issue #105](aspnet#105) so that people do not have to go hunting it down to get the new module.

This should make it easier to get the word out about the module, and reduce the time people spend hunting it down in the _Issues_ section.

### CODE REVIEW

**Comment by Tratcher** [comment]
The content of the announcement is transitional, where the readme should discuss current usage.  Let's split the difference and add the following to the readme:
- A brief description of what AspNetCoreModule is used for and the download links.
- A link to the announcement under a heading "Migrating from HttpPlatformHandler".

**Comment by ciel** [comment]
Sure. I've made the edits you suggested, please tell me what you think.

File: README.md [hunk 1]
```diff
@@ -4,3 +4,17 @@ ASP.NET IIS Integration
 This repo hosts the ASP.NET Core middleware for IIS integration.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+# ASP.NET Core Module
+
+The ASP.NET Core Module is superseding the IISPlatformHandler module in IIS configurations.
```
Comment (Tratcher) [inline README.md]: IISPlatformHandler -> HttpPlatformHandler

**Comment by Tratcher** [comment]
@muratg?

**Comment by ciel** [comment]
Oops, let me fix that real quick.

**Comment by ciel** [comment]
Haha, there we go.

**Comment by muratg** [comment]
LGTM

**Comment by Tratcher** [comment]
I talked to the team and we decided that we don't want the download links for v0.8 in the dev or rc2 readme. v0.8 was just a preview to gather feedback and help with the transition, there will be a new installer released with RC2 for general consumption, and updates on docs.asp.net covering the usage and migration. We're prepping the RC2 release now, so you shouldn't need to wait too much longer.

**Comment by ciel** [comment]
Awesome. I'm so excited for the big push of RC2!

**Pull request closed without merging**